cvmfs/cvmfs_server

```diff
@@ -2055,6 +2055,13 @@ Alias /cvmfs/${name} ${storage_dir}
     ExpiresByType application/x-cvmfs "access plus 2 minutes"
     ExpiresByType application/json    "access plus 2 minutes"
 </Directory>
+
+# Load http2 module
+LoadModule http2_module modules/mod_http2.so
```

Comment:
it won't, but this module should be available if using a sufficiently new apache version, which (I guess) is intended to happen. In any case it would be possible to check the installed modules in advance, and only if the module is present then it could be loaded